libnetwork/internal/kvstore/boltdb/boltdb.go

```diff
@@ -19,6 +19,8 @@ var (
 	ErrBoltPathOptionMissing = errors.New("path to the boltdb is missing")
 	// ErrBoltBucketOptionMissing is returned when the bucket name is empty
 	ErrBoltBucketOptionMissing = errors.New("boltdb's bucket name is missing")
+	// ErrDBAlreadyOpen signals that this boltdb is already open by another
+	ErrDBAlreadyOpen = errors.New("the underlying boltdb file is already held open")
```

Comment:
See my previous comment; if this is not used as a sentinel error, we shouldn't export it, and just inline the error.